src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/doclets_ja.properties

```diff
@@ -278,9 +278,9 @@ doclet.record_constructor_doc.param_name={0}レコード・コンポーネント
 
 doclet.record_equals_doc.fullbody.head=他のオブジェクトがこれと"等しい"かどうかを示します。他のオブジェクトが同じクラスであり、すべてのレコード・コンポーネントが等しい場合、オブジェクトは等しくなります。
 
-doclet.record_equals_doc.fullbody.tail.both=参照コンポーネントは{@link java.util.Objects#equals(Object,Object) Objects::equals(Object,Object)}と比較され、プリミティブ・コンポーネントは'=='と比較されます。
+doclet.record_equals_doc.fullbody.tail.both=参照コンポーネントは{@link java.util.Objects#equals(Object,Object) Objects::equals(Object,Object)}と比較され、プリミティブ・コンポーネントは対応するラッパー・クラスの<code>compare</code>メソッドと比較されます。
 
-doclet.record_equals_doc.fullbody.tail.primitive=このレコード・クラスのすべてのコンポーネントは'=='と比較されます。
+doclet.record_equals_doc.fullbody.tail.primitive=このレコード・クラスのすべてのコンポーネントは対応するラッパー・クラスの<code>compare</code>メソッドと比較されます。
```

Comment:
These translation are somewhat questionable. IIUC the original English meaning is to compare components to each other *using* the `compare` methods in the wrappers, but these Japanese translations imply compare components *to* the `compare` methods in the wrappers.

Reply:
If you are able to provide a better translation, that would be ideal. We can replace this translation with that one, as well as filing a bug against the translation tool asking them to use yours.

Otherwise, we should probably not revert this change, as while it is questionable, it is still updated to match the new English definition, and the previous localized definition would not match since it is outdated.

Reply:
Cannot suggest the translation for the second line on GitHub (complaining "Applying suggestions on deleted lines is not supported."), but the same diff can be applied here

Reply:
> Cannot suggest the translation for the second line on GitHub (complaining "Applying suggestions on deleted lines is not supported."), but the same diff can be applied here

I think I handled the issue you mentioned here. I replaced line 283 with the same suggested change you commented on for line 281. Let me know if there are any issues, but if this is correct, the current state of the PR should have fixed both of your concerns. Thanks!

Reply:
Thanks, Damon. Now the latest revision has the correct translation.